LPD-60283 Shared with Me: custom cell renderers for Title and Shared By cells

ambrinchaudhary opened the pull request:
Link to issue: https://liferay.atlassian.net/browse/LPD-60283

This is how it will look like once fileTypeIcon and fileTypeIconColor are send for each team. Also, I'm guessing the siteName is the space where it has been shared from. 

<img width="1253" height="141" alt="Screenshot 2025-07-16 at 12 51 10" src="https://github.com/user-attachments/assets/4faa1644-e2f4-44e4-83d0-6d2bf4b68072" />

TODO: link to view the items. 

cc/ @AliciaGarciaGarcia

liferay-continuous-integration commented:
Test suite sf has been triggered on http://test-1-29

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 60 seconds</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/e7cde4617ed5397f6aed1f82ce207931a3d2477e">e7cde4617ed5397f6aed1f82ce207931a3d2477e</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/ambrinchaudhary/liferay-portal/tree/LPD-60283_SharedWithMe_CellRenderer">LPD-60283_SharedWithMe_CellRenderer</a><br/>Branch GIT ID: <a href="https://github.com/ambrinchaudhary/liferay-portal/commit/616bfe9ebf77ec3f116c37a4e27870355c19419e">616bfe9ebf77ec3f116c37a4e27870355c19419e</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-29.liferay.com/job/test-portal-source-format/6513/"><strike>test-portal-source-format</strike></a></li></ul></div><h5>For more details click <a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-source-format/builds/6513/jenkins-report.html">here</a>.</h5><div><p>Please fix <strong>rebase errors</strong> on <strong><a href="https://github.com/liferay-content-management/liferay-portal/pull/6860">ambrinchaudhary/LPD-60283_SharedWithMe_CellRenderer</a></strong><pre>Unable to rebase master-temp-1752664659495 to master
Rebasing (1/13)
Rebasing (2/13)
Rebasing (3/13)
error: could not apply f9492fb35404c... LPD-55387 show the empty message with the new image</pre></p></div></details></html>

liferay-continuous-integration commented:
Test suite sf has been triggered on http://test-1-40

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-40.liferay.com/job/test-portal-source-format/8914/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-source-format/builds/8914/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-40.liferay.com/job/test-portal-source-format/8914">test-portal-source-format#8914</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-source-format/builds/8914/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-content-management/liferay-portal/pull/6860">liferay-content-management#6860</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/262236034">[master] ci:test:sf - ambrinchaudhary &gt; liferay-content-management - PR#6860 - 2025-07-16[04:34:24]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>262236034</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-40.liferay.com/job/test-portal-source-format/8914/">test-portal-source-format#8914</a><br/></div></div>

liferay-continuous-integration commented:
Test suite sf has been triggered on http://test-1-36

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-36.liferay.com/job/test-portal-source-format/4581/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-36.liferay.com/userContent/jobs/test-portal-source-format/builds/4581/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 4 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/41b10d0dc3f330aa667110fd0d3b606abee88058">41b10d0dc3f330aa667110fd0d3b606abee88058</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/ambrinchaudhary/liferay-portal/tree/LPD-60283_SharedWithMe_CellRenderer">LPD-60283_SharedWithMe_CellRenderer</a><br/>Branch GIT ID: <a href="https://github.com/ambrinchaudhary/liferay-portal/commit/492f6e536fc9fcea7f9240c302047217eab6a4de">492f6e536fc9fcea7f9240c302047217eab6a4de</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-36.liferay.com/job/test-portal-source-format/4581/"><strike>test-portal-source-format</strike></a></li></ul></div><h5>For more details click <a href="https://test-1-36.liferay.com/userContent/jobs/test-portal-source-format/builds/4581/jenkins-report.html">here</a>.</h5><pre>format-source-files:
     [java] java.lang.Exception: Found 1 formatting issues:
     [java] 1: ./modules/apps/site/site-cms-site-initializer/test.properties expected:&lt;[modified.files.includes[relevant][site-cms-site-initializer-playwright-rule]=**
     [java] 
     [java] playwright.projects.includes[playwright-js-tomcat101-mysql57][relevant][site-cms-site-initializer-playwright-rule]=\
     [java]     site-cms-site-initializer.main
     [java] 
     [java] relevant.rule.names=site-cms-site-initializer-playwright-rule
     [java] 
     [java] test.batch.names[relevant][site-cms-site-initializer-playwright-rule]=\
     [java]     playwright-js-tomcat101-mysql57
     [java] 
     [java] ]testray.main.compone...&gt; but was:&lt;[##
     [java] ## Test Suites
     [java] ##
     [java] 
     [java]     modified.files.includes[relevant][site-cms-site-initializer-playwright-rule]=**
     [java]     relevant.rule.names=site-cms-site-initializer-playwright-rule
     [java] 
     [java]     test.batch.names[relevant][site-cms-site-initializer-playwright-rule]=\
     [java]         playwright-js-tomcat101-mysql57
     [java] 
     [java]     #
     [java]     # Relevant
     [java]     #
     [java] 
     [java]     playwright.projects.includes[playwright-js-tomcat101-mysql57][relevant][site-cms-site-initializer-playwright-rule]=\
     [java]         site-cms-site-initializer.main
     [java] 
     [java] ##
     [java] ## Testray
     [java] ##
     [java] 
     [java]     ]testray.main.compone...&gt;
     [java] 
     [java]   at com.liferay.source.formatter.SourceFormatter.format(SourceFormatter.java:474)
     [java]   at com.liferay.source.formatter.SourceFormatter.main(SourceFormatter.java:302)
[stopwatch] [run.batch.test.action: 1:46.312 sec]
     [echo] The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/portal-impl/build.xml:471: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/portal-impl/build.xml:624: Java returned: 1
  [typedef] Could not load definitions from resource org/apache/maven/artifact/ant/antlib.xml. It could not be found.
  [taskdef] Could not load definitions from resource org/jacoco/ant/antlib.xml. It could not be found.
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null2113367134.properties

merge-test-results:
    [mkdir] Created dir: /opt/dev/projects/github/liferay-portal/test-results</pre></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-36.liferay.com/job/test-portal-source-format/4581">test-portal-source-format#4581</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-36.liferay.com/userContent/jobs/test-portal-source-format/builds/4581/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-content-management/liferay-portal/pull/6860">liferay-content-management#6860</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/262323383">[master] ci:test:sf - ambrinchaudhary &gt; liferay-content-management - PR#6860 - 2025-07-16[06:01:59]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>262323383</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-36.liferay.com/job/test-portal-source-format/4581/">test-portal-source-format#4581</a><br/></div></div>

ambrinchaudhary commented:
I changed and added it accidentaly on https://github.com/liferay-content-management/liferay-portal/pull/6860/commits/4e9afa6c79a1cc0524a92a11c0871d669e761ef6#diff-b3b6d5e84ba61d6ce3a03344d80cd8789c80f44ff825050fd9b041593959adaa, let me fix it....

ambrinchaudhary commented:
ci:test:sf

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-29.liferay.com/job/test-portal-source-format/6542/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-source-format/builds/6542/jenkins-report.html">here</a>.</b></p>

AliciaGarciaGarcia commented:
ci:test:sf

liferay-continuous-integration commented:
Test suite sf has been triggered on http://test-1-37

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 3 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/f93d812d33c411087fcd1965824fe1479a8629b1">f93d812d33c411087fcd1965824fe1479a8629b1</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/ambrinchaudhary/liferay-portal/tree/LPD-60283_SharedWithMe_CellRenderer">LPD-60283_SharedWithMe_CellRenderer</a><br/>Branch GIT ID: <a href="https://github.com/ambrinchaudhary/liferay-portal/commit/45d230b7ded99dc25b12026cfff6ac5f29b0a9fb">45d230b7ded99dc25b12026cfff6ac5f29b0a9fb</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-37.liferay.com/job/test-portal-source-format/13580/"><strike>test-portal-source-format</strike></a></li></ul></div><h5>For more details click <a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-source-format/builds/13580/jenkins-report.html">here</a>.</h5><pre>format-source-files:
     [java] java.lang.Exception: Found 1 formatting issues:
     [java] 1: ./modules/apps/site/site-cms-site-initializer/test.properties expected:&lt;[modified.files.includes[relevant][site-cms-site-initializer-playwright-rule]=**
     [java] 
     [java] playwright.projects.includes[playwright-js-tomcat101-mysql57][relevant][site-cms-site-initializer-playwright-rule]=\
     [java]     site-cms-site-initializer.main
     [java] 
     [java] relevant.rule.names=site-cms-site-initializer-playwright-rule
     [java] 
     [java] test.batch.names[relevant][site-cms-site-initializer-playwright-rule]=\
     [java]     playwright-js-tomcat101-mysql57
     [java] 
     [java] ]testray.main.compone...&gt; but was:&lt;[##
     [java] ## Test Suites
     [java] ##
     [java] 
     [java]     modified.files.includes[relevant][site-cms-site-initializer-playwright-rule]=**
     [java]     relevant.rule.names=site-cms-site-initializer-playwright-rule
     [java] 
     [java]     test.batch.names[relevant][site-cms-site-initializer-playwright-rule]=\
     [java]         playwright-js-tomcat101-mysql57
     [java] 
     [java]     #
     [java]     # Relevant
     [java]     #
     [java] 
     [java]     playwright.projects.includes[playwright-js-tomcat101-mysql57][relevant][site-cms-site-initializer-playwright-rule]=\
     [java]         site-cms-site-initializer.main
     [java] 
     [java] ##
     [java] ## Testray
     [java] ##
     [java] 
     [java]     ]testray.main.compone...&gt;
     [java] 
     [java]   at com.liferay.source.formatter.SourceFormatter.format(SourceFormatter.java:474)
     [java]   at com.liferay.source.formatter.SourceFormatter.main(SourceFormatter.java:302)
[stopwatch] [run.batch.test.action: 1:31.130 sec]
     [echo] The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/portal-impl/build.xml:471: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/portal-impl/build.xml:624: Java returned: 1
  [typedef] Could not load definitions from resource org/apache/maven/artifact/ant/antlib.xml. It could not be found.
  [taskdef] Could not load definitions from resource org/jacoco/ant/antlib.xml. It could not be found.
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null1898883007.properties

merge-test-results:
    [mkdir] Created dir: /opt/dev/projects/github/liferay-portal/test-results</pre></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-37.liferay.com/job/test-portal-source-format/13580">test-portal-source-format#13580</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-source-format/builds/13580/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-content-management/liferay-portal/pull/6860">liferay-content-management#6860</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/266998490">[master] ci:test:sf - ambrinchaudhary &gt; liferay-content-management - PR#6860 - 2025-07-23[08:05:54]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>266998490</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-37.liferay.com/job/test-portal-source-format/13580/">test-portal-source-format#13580</a><br/></div></div>

AliciaGarciaGarcia commented:
ci:test:sf

liferay-continuous-integration commented:
Test suite sf has been triggered on http://test-1-31

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-31.liferay.com/job/test-portal-source-format/7319/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-31.liferay.com/userContent/jobs/test-portal-source-format/builds/7319/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-31.liferay.com/job/test-portal-source-format/7319">test-portal-source-format#7319</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-31.liferay.com/userContent/jobs/test-portal-source-format/builds/7319/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-content-management/liferay-portal/pull/6860">liferay-content-management#6860</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/267027227">[master] ci:test:sf - ambrinchaudhary &gt; liferay-content-management - PR#6860 - 2025-07-23[08:28:34]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>267027227</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-31.liferay.com/job/test-portal-source-format/7319/">test-portal-source-format#7319</a><br/></div></div>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-40.liferay.com/job/test-portal-source-format/8976/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-source-format/builds/8976/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

The pull request will automatically be forwarded to the user `brianchandotcom` If the following test suites pass:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

AliciaGarciaGarcia commented:
ci:reevaluate:1367401_8626

liferay-continuous-integration commented:
CI is reevaluating the build with build ID: `1367401_8626` against the latest valid upstream results.

liferay-continuous-integration commented:
Test suite default has been triggered on http://test-1-31

AliciaGarciaGarcia commented:
ci:forward

liferay-continuous-integration commented:
Test suite relevant has been triggered on http://test-1-39

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest(master)/14291/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-39.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/14291/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:stable - 11 out of 12 jobs passed</h3><h3>:x: ci:test:relevant - 15 out of 19 jobs passed in 1 hour 17 minutes</h3><details><summary>Click here for more details.</summary><br/><p>This pull is eligible for <a href="https://grow.liferay.com/share/CI+liferay-continuous-integration+GitHub+Commands#General-Commands">reevaluation</a>. When this <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)/1551">upstream build</a> has completed, using the following CI command will compare this pull request result against a more recent upstream result:<pre>ci:reevaluate:1397401_14291</pre></p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/5f55dcced394bd432b0b75bba6c9035fd17c57ca">5f55dcced394bd432b0b75bba6c9035fd17c57ca</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/feebebc14c34f564345bbb2b86d96ad7863daf32">feebebc14c34f564345bbb2b86d96ad7863daf32</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/#/project/35392/routines/590307/build/271143012">EE Development Acceptance (master) - 1550 - 2025-07-28[12:33:40]</a></p><details><summary><strong>ci:test:stable - 11 out of 12 jobs PASSED</strong></summary><div><h4>1 Failed Jobs:</h4><ul><li>central-requirements<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1191261/">central-requirements/0/0</a></li></ul></li><li>functional-upgrade-tomcat101-postgresql163_stable<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1403955/">functional-upgrade-tomcat101-postgresql163_stable/0/0</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1228875/">functional-upgrade-tomcat101-postgresql163_stable/0/1</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1103740/">functional-upgrade-tomcat101-postgresql163_stable/0/2</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1171902/">functional-upgrade-tomcat101-postgresql163_stable/0/3</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1290697/">functional-upgrade-tomcat101-postgresql163_stable/0/4</a></li></ul></li><li>js-unit<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1228870/">js-unit/0/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1290693/"><strike>js-unit/1/0</strike></a></li></ul></li><li>modules-compile<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1223337/">modules-compile/0/0</a></li></ul></li><li>modules-integration-postgresql163_stable<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1191269/">modules-integration-postgresql163_stable/0/0</a></li></ul></li><li>playwright-js-tomcat101-postgresql163_stable<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1191270/">playwright-js-tomcat101-postgresql163_stable/0/0</a></li></ul></li><li>unit_stable<ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1171901/">unit_stable/0/0</a></li></ul></li></ul></div><details><summary><strong>11 Successful Jobs:</strong></summary><ul/></details></details><details><summary><strong>ci:test:relevant - 15 out of 19 jobs PASSED</strong></summary><div><h4>4 Failed Jobs:</h4><ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest(master)/14291/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>central-requirements<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1191261/">central-requirements/0/0</a></li></ul></li><li>functional-upgrade-tomcat101-postgresql163_stable<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1403955/">functional-upgrade-tomcat101-postgresql163_stable/0/0</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1228875/">functional-upgrade-tomcat101-postgresql163_stable/0/1</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1103740/">functional-upgrade-tomcat101-postgresql163_stable/0/2</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1171902/">functional-upgrade-tomcat101-postgresql163_stable/0/3</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1290697/">functional-upgrade-tomcat101-postgresql163_stable/0/4</a></li></ul></li><li>js-unit<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1228870/">js-unit/0/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1290693/"><strike>js-unit/1/0</strike></a></li></ul></li><li>modules-compile<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1223337/">modules-compile/0/0</a></li></ul></li><li>modules-integration-postgresql163<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1223338/"><strike>modules-integration-postgresql163/0/0</strike></a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1191272/">modules-integration-postgresql163/0/1</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1191271/">modules-integration-postgresql163/0/2</a></li></ul></li><li>modules-integration-postgresql163_stable<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1191269/">modules-integration-postgresql163_stable/0/0</a></li></ul></li><li>modules-semantic-versioning<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1191262/">modules-semantic-versioning/0/0</a></li></ul></li><li>modules-unit<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1228874/">modules-unit/0/0</a></li></ul></li><li>playwright-js-tomcat101-mysql57<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1223335/"><strike>playwright-js-tomcat101-mysql57/0/0</strike></a></li></ul></li><li>playwright-js-tomcat101-postgresql163_stable<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1191270/">playwright-js-tomcat101-postgresql163_stable/0/0</a></li></ul></li><li>unit_stable<ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1171901/">unit_stable/0/0</a></li></ul></li></ul></div><details><summary><strong>15 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-39.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/14291/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1290693//consoleText">js-unit/1/0</a><ol><li><details><summary><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1290693/testReport/(root)/:apps:commerce:commerce-frontend-js:packageRunTest">:apps:commerce:commerce-frontend-js:packageRunTest</a></summary><div><ol><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1290693//testReport/(root)/:apps:commerce:commerce-frontend-js:packageRunTest/apps_commerce_commerce_frontend_js_test_components_AddressSelector_AddressSelector_Must_display_error_message_if_address_creation_API_is_not_called_correctly">apps.commerce.commerce-frontend-js.test.components.AddressSelector.AddressSelector Must display error message if address creation API is not called correctly</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1290693//testReport/(root)/:apps:commerce:commerce-frontend-js:packageRunTest/apps_commerce_commerce_frontend_js_test_components_AddressSelector_AddressSelector_Must_hide_the_subtype_field_if_configuration_is_not_provided">apps.commerce.commerce-frontend-js.test.components.AddressSelector.AddressSelector Must hide the subtype field if configuration is not provided</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1290693//testReport/(root)/:apps:commerce:commerce-frontend-js:packageRunTest/apps_commerce_commerce_frontend_js_test_components_AddressSelector_AddressSelector_Must_preload_subtype_field_if_specified_address_as_it">apps.commerce.commerce-frontend-js.test.components.AddressSelector.AddressSelector Must preload subtype field if specified address as it</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1290693//testReport/(root)/:apps:commerce:commerce-frontend-js:packageRunTest/apps_commerce_commerce_frontend_js_test_components_AddressSelector_AddressSelector_Must_preload_the_subtype_field_when_selecting_an_existing_address">apps.commerce.commerce-frontend-js.test.components.AddressSelector.AddressSelector Must preload the subtype field when selecting an existing address</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1290693//testReport/(root)/:apps:commerce:commerce-frontend-js:packageRunTest/apps_commerce_commerce_frontend_js_test_components_AddressSelector_AddressSelector_Must_save_an_empty_subtype_field_when_adding_a_new_address">apps.commerce.commerce-frontend-js.test.components.AddressSelector.AddressSelector Must save an empty subtype field when adding a new address</a></li><li>...</li></ol></div></details></li><li><details><summary><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1290693/testReport/(root)/:apps:layout:layout-content-page-editor-web:packageRunTest">:apps:layout:layout-content-page-editor-web:packageRunTest</a></summary><div><ol><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1290693//testReport/(root)/:apps:layout:layout-content-page-editor-web:packageRunTest/apps_layout_layout_content_page_editor_web_test_page_editor_app_components_fragment_configuration_fields_CSSClassSelectorField_add_cssClass_when_clicking_modal_button">apps.layout.layout-content-page-editor-web.test.page_editor.app.components.fragment_configuration_fields.CSSClassSelectorField add cssClass when clicking modal button</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1290693//testReport/(root)/:apps:layout:layout-content-page-editor-web:packageRunTest/apps_layout_layout_content_page_editor_web_test_page_editor_app_components_fragment_configuration_fields_CSSClassSelectorField_adds_classes_with_comma__enter_and_space">apps.layout.layout-content-page-editor-web.test.page_editor.app.components.fragment_configuration_fields.CSSClassSelectorField adds classes with comma, enter and space</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1290693//testReport/(root)/:apps:layout:layout-content-page-editor-web:packageRunTest/apps_layout_layout_content_page_editor_web_test_page_editor_app_components_fragment_configuration_fields_CSSClassSelectorField_autocomplete_with_classNames_of_other_components">apps.layout.layout-content-page-editor-web.test.page_editor.app.components.fragment_configuration_fields.CSSClassSelectorField autocomplete with classNames of other components</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1290693//testReport/(root)/:apps:layout:layout-content-page-editor-web:packageRunTest/apps_layout_layout_content_page_editor_web_test_page_editor_app_components_fragment_configuration_fields_CSSClassSelectorField_calls_onValueSelect_when_selecting_a_new_class">apps.layout.layout-content-page-editor-web.test.page_editor.app.components.fragment_configuration_fields.CSSClassSelectorField calls onValueSelect when selecting a new class</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1290693//testReport/(root)/:apps:layout:layout-content-page-editor-web:packageRunTest/apps_layout_layout_content_page_editor_web_test_page_editor_app_components_fragment_configuration_fields_CSSClassSelectorField_shows_modal_with_create_option">apps.layout.layout-content-page-editor-web.test.page_editor.app.components.fragment_configuration_fields.CSSClassSelectorField shows modal with create option</a></li><li>...</li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1223338//consoleText">modules-integration-postgresql163/0/0</a><ol><li><details><summary><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1223338/testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest">com.liferay.portal.log.assertor.PortalLogAssertorTest</a></summary><div><ol><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1223338//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">testScanXMLLog</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1223335//consoleText">playwright-js-tomcat101-mysql57/0/0</a> - <a href="https://storage.cloud.google.com/testray-results/2025-07/test-1-39/test-portal-acceptance-pullrequest(master)/14291/playwright-js-tomcat101-mysql57/0/0/playwright-report/index.html">Playwright Report</a><ol><li><details><summary><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1223335/testReport/site-cms-site-initializer_main_files.spec/ts">site-cms-site-initializer/main/files.spec.ts</a></summary><div><ol><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1223335//testReport/site-cms-site-initializer_main_files.spec/ts/The_Space_selector_dialog_is_not_shown_when_creating_a_Basic_Document_inside_a_folder_when_multiple_Spaces_exist">The Space selector dialog is not shown when creating a Basic Document inside a folder when multiple Spaces exist</a></li></ol></div></details></li><li><details><summary><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1223335/testReport/site-cms-site-initializer_main_folders.spec/ts">site-cms-site-initializer/main/folders.spec.ts</a></summary><div><ol><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1223335//testReport/site-cms-site-initializer_main_folders.spec/ts/Can_edit_a_folder">Can edit a folder</a></li></ol></div></details></li><li><details><summary><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1223335/testReport/site-cms-site-initializer_main_contentEditor.spec/ts">site-cms-site-initializer/main/contentEditor.spec.ts</a></summary><div><ol><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1223335//testReport/site-cms-site-initializer_main_contentEditor.spec/ts/When_publishing_a_content_in_a_folder_the_browser_is_redirected_to_the_folder">When publishing a content in a folder the browser is redirected to the folder</a></li></ol></div></details></li><li>...</li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

ambrinchaudhary commented:
ci:report:271461689

ambrinchaudhary commented:
[Can edit a folder](https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1223335//testReport/site-cms-site-initializer_main_folders.spec/ts/Can_edit_a_folder)
This error is on master, not related to this PR

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running <code>generate testray csv</code> for testrayBuildID: <b>271461689</b>.</p><p>Job Link: <a href="https://test-1-34.liferay.com/job/generate-testray-csv/321/">generate-testray-csv</a></p>

AliciaGarciaGarcia commented:
ci:forward

liferay-continuous-integration commented:
Test suite relevant has been triggered on http://test-1-36

liferay-continuous-integration commented:
<html><h3>:x: ci:test:stable - 11 out of 12 jobs passed</h3><h3>:x: ci:test:relevant - 15 out of 19 jobs passed in 1 hour 51 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/9af93b197cc489139dc99822ec6242fc0465c43e">9af93b197cc489139dc99822ec6242fc0465c43e</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/9af93b197cc489139dc99822ec6242fc0465c43e">9af93b197cc489139dc99822ec6242fc0465c43e</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/#/project/35392/routines/590307/build/273532844">EE Development Acceptance (master) - 1556 - 2025-07-30[16:33:31]</a></p><details><summary><strong>ci:test:stable - 11 out of 12 jobs PASSED</strong></summary><div><h4>1 Failed Jobs:</h4><ul><li>central-requirements<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1224975/">central-requirements/0/0</a></li></ul></li><li>functional-upgrade-tomcat101-postgresql163_stable<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1224982/">functional-upgrade-tomcat101-postgresql163_stable/0/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1325231/">functional-upgrade-tomcat101-postgresql163_stable/0/1</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1261849/">functional-upgrade-tomcat101-postgresql163_stable/0/2</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1292495/">functional-upgrade-tomcat101-postgresql163_stable/0/3</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1230717/">functional-upgrade-tomcat101-postgresql163_stable/0/4</a></li></ul></li><li>js-unit<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1325223/"><strike>js-unit/0/0</strike></a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1173557/">js-unit/1/0</a></li></ul></li><li>modules-compile<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1105320/">modules-compile/0/0</a></li></ul></li><li>modules-integration-postgresql163_stable<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/877047/">modules-integration-postgresql163_stable/0/0</a></li></ul></li><li>playwright-js-tomcat101-postgresql163_stable<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1300456/">playwright-js-tomcat101-postgresql163_stable/0/0</a></li></ul></li><li>unit_stable<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1224981/">unit_stable/0/0</a></li></ul></li></ul></div><details><summary><strong>11 Successful Jobs:</strong></summary><ul/></details></details><details><summary><strong>ci:test:relevant - 15 out of 19 jobs PASSED</strong></summary><div><h4>4 Failed Jobs:</h4><ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest(master)/8677/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>central-requirements<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1224975/">central-requirements/0/0</a></li></ul></li><li>functional-upgrade-tomcat101-postgresql163_stable<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1224982/">functional-upgrade-tomcat101-postgresql163_stable/0/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1325231/">functional-upgrade-tomcat101-postgresql163_stable/0/1</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1261849/">functional-upgrade-tomcat101-postgresql163_stable/0/2</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1292495/">functional-upgrade-tomcat101-postgresql163_stable/0/3</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1230717/">functional-upgrade-tomcat101-postgresql163_stable/0/4</a></li></ul></li><li>js-unit<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1325223/"><strike>js-unit/0/0</strike></a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1173557/">js-unit/1/0</a></li></ul></li><li>modules-compile<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1105320/">modules-compile/0/0</a></li></ul></li><li>modules-integration-postgresql163<ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1475163/"><strike>modules-integration-postgresql163/0/0</strike></a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1300457/">modules-integration-postgresql163/0/1</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1325230/">modules-integration-postgresql163/0/2</a></li></ul></li><li>modules-integration-postgresql163_stable<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/877047/">modules-integration-postgresql163_stable/0/0</a></li></ul></li><li>modules-semantic-versioning<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1300453/">modules-semantic-versioning/0/0</a></li></ul></li><li>modules-unit<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/877048/">modules-unit/0/0</a></li></ul></li><li>playwright-js-tomcat101-mysql57<ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1261850/"><strike>playwright-js-tomcat101-mysql57/0/0</strike></a></li></ul></li><li>playwright-js-tomcat101-postgresql163_stable<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1300456/">playwright-js-tomcat101-postgresql163_stable/0/0</a></li></ul></li><li>unit_stable<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1224981/">unit_stable/0/0</a></li></ul></li></ul></div><details><summary><strong>15 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-36.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/8677/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1475163//consoleText">modules-integration-postgresql163/0/0</a><ol><li><details><summary><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1475163/testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest">com.liferay.portal.log.assertor.PortalLogAssertorTest</a></summary><div><ol><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1475163//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">testScanXMLLog</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1261850//consoleText">playwright-js-tomcat101-mysql57/0/0</a> - <a href="https://storage.cloud.google.com/testray-results/2025-07/test-1-36/test-portal-acceptance-pullrequest(master)/8677/playwright-js-tomcat101-mysql57/0/0/playwright-report/index.html">Playwright Report</a><ol><li><details><summary><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1261850/testReport/site-cms-site-initializer_main_files.spec/ts">site-cms-site-initializer/main/files.spec.ts</a></summary><div><ol><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1261850//testReport/site-cms-site-initializer_main_files.spec/ts/The_Space_selector_dialog_is_not_shown_when_creating_a_Basic_Document_inside_a_folder_when_multiple_Spaces_exist">The Space selector dialog is not shown when creating a Basic Document inside a folder when multiple Spaces exist</a></li></ol></div></details></li><li><details><summary><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1261850/testReport/site-cms-site-initializer_main_contentEditor.spec/ts">site-cms-site-initializer/main/contentEditor.spec.ts</a></summary><div><ol><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1261850//testReport/site-cms-site-initializer_main_contentEditor.spec/ts/When_publishing_a_content_in_a_folder_the_browser_is_redirected_to_the_folder">When publishing a content in a folder the browser is redirected to the folder</a></li></ol></div></details></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

liferay-continuous-integration commented:
Test suite default has been triggered on http://test-1-29

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running <code>generate testray csv</code> for testrayBuildID: <b>273742195</b>.</p><p>Job Link: <a href="https://test-1-29.liferay.com/job/generate-testray-csv/502/">generate-testray-csv</a></p>

liferay-continuous-integration commented:
Skipping previously passed test suites:
- `ci:test:sf`

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 12 out of 12 jobs passed</h3><h3>:x: ci:test:relevant - 17 out of 20 jobs passed in 1 hour 28 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/c895faf9ae900707365c49e78e29bd3025f5cdc9">c895faf9ae900707365c49e78e29bd3025f5cdc9</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/c895faf9ae900707365c49e78e29bd3025f5cdc9">c895faf9ae900707365c49e78e29bd3025f5cdc9</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/#/project/35392/routines/590307/build/274918389">EE Development Acceptance (master) - 1557 - 2025-07-31[20:39:48]</a></p><details><summary><strong>ci:test:stable - 12 out of 12 jobs PASSED</strong></summary><details><summary><strong>12 Successful Jobs:</strong></summary><ul/></details></details><details><summary><strong>ci:test:relevant - 17 out of 20 jobs PASSED</strong></summary><div><h4>3 Failed Jobs:</h4><ul><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest(master)/12396/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>central-requirements<ul><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/992902/">central-requirements/0/0</a></li></ul></li><li>functional-upgrade-tomcat101-postgresql163_stable<ul><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1118410/">functional-upgrade-tomcat101-postgresql163_stable/0/0</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/877459/">functional-upgrade-tomcat101-postgresql163_stable/0/1</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1174012/">functional-upgrade-tomcat101-postgresql163_stable/0/2</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1325721/">functional-upgrade-tomcat101-postgresql163_stable/0/3</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1262248/">functional-upgrade-tomcat101-postgresql163_stable/0/4</a></li></ul></li><li>js-unit<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1231333/">js-unit/0/0</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/992901/">js-unit/1/0</a></li></ul></li><li>modules-compile<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1231338/">modules-compile/0/0</a></li></ul></li><li>modules-integration-postgresql163<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1231348/"><strike>modules-integration-postgresql163/0/0</strike></a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1475705/">modules-integration-postgresql163/0/1</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1225517/">modules-integration-postgresql163/0/2</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1105797/">modules-integration-postgresql163/0/3</a></li></ul></li><li>modules-integration-postgresql163_stable<ul><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1118409/">modules-integration-postgresql163_stable/0/0</a></li></ul></li><li>modules-semantic-versioning<ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1475701/">modules-semantic-versioning/0/0</a></li></ul></li><li>modules-unit<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1194199/">modules-unit/0/0</a></li></ul></li><li>playwright-js-tomcat101-mysql57<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1300794/"><strike>playwright-js-tomcat101-mysql57/0/0</strike></a></li></ul></li><li>playwright-js-tomcat101-postgresql163_stable<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1300793/">playwright-js-tomcat101-postgresql163_stable/0/0</a></li></ul></li><li>unit_stable<ul><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/992907/">unit_stable/0/0</a></li></ul></li></ul></div><details><summary><strong>17 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/12396/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1231348//consoleText">modules-integration-postgresql163/0/0</a><ol><li><details><summary><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1231348/testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest">com.liferay.portal.log.assertor.PortalLogAssertorTest</a></summary><div><ol><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1231348//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">testScanXMLLog</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1300794//consoleText">playwright-js-tomcat101-mysql57/0/0</a> - <a href="https://storage.cloud.google.com/testray-results/2025-07/test-1-29/test-portal-acceptance-pullrequest(master)/12396/playwright-js-tomcat101-mysql57/0/0/playwright-report/index.html">Playwright Report</a><ol><li><details><summary><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1300794/testReport/site-cms-site-initializer_main_files.spec/ts">site-cms-site-initializer/main/files.spec.ts</a></summary><div><ol><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1300794//testReport/site-cms-site-initializer_main_files.spec/ts/The_Space_selector_dialog_is_not_shown_when_creating_a_Basic_Document_inside_a_folder_when_multiple_Spaces_exist">The Space selector dialog is not shown when creating a Basic Document inside a folder when multiple Spaces exist</a></li></ol></div></details></li><li><details><summary><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1300794/testReport/site-cms-site-initializer_main_contentEditor.spec/ts">site-cms-site-initializer/main/contentEditor.spec.ts</a></summary><div><ol><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1300794//testReport/site-cms-site-initializer_main_contentEditor.spec/ts/When_publishing_a_content_in_a_folder_the_browser_is_redirected_to_the_folder">When publishing a content in a folder the browser is redirected to the folder</a></li></ol></div></details></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest(master)/12396">test-portal-acceptance-pullrequest(master)#12396</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/12396/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-content-management/liferay-portal/pull/6860">liferay-content-management#6860</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/275073683">[master] ci:test:relevant - ambrinchaudhary &gt; liferay-content-management - PR#6860 - 2025-07-31[23:35:31]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>275073683</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest(master)/12396/">test-portal-acceptance-pullrequest(master)#12396</a><br/></div></div>

ambrinchaudhary commented:
ci:forward:force

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

The pull request will automatically be forwarded to the user `brianchandotcom` if the following test suites complete:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
AND If the following test suites pass:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

liferay-continuous-integration commented:
Skipping previously completed test suites:
- `ci:test:relevant`
- `ci:test:sf`
AND Skipping previously passed test suites:
- `ci:test:stable`

liferay-continuous-integration commented:
Test suite default has been triggered on http://test-1-36

liferay-continuous-integration commented:
All required test suite(s) completed.
Forwarding pull request to `brianchandotcom`.
[Console](https://test-1-36.liferay.com/job/forward-pullrequest/1013//consoleFull)

liferay-continuous-integration commented:
Pull request has been successfully forwarded to  https://github.com/brianchandotcom/liferay-portal/pull/164133
[Console](https://test-1-36.liferay.com/job/forward-pullrequest/1013//consoleFull)
